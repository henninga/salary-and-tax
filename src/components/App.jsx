import React from 'react';
import { SalaryCalculations } from './SalaryCalculations';
import { SalaryParameters } from './SalaryParameters';


export const AppComponent = () => (
  <div>
    <SalaryCalculations />
    <SalaryParameters />
  </div>
);

export default AppComponent;
