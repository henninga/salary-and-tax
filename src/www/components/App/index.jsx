import PropTypes from 'prop-types';
import React from 'react';
import { connect } from 'react-redux';

export const AppComponent = ({ name }) => (
  <div>
    { `Hello ${name}` }
  </div>
);

AppComponent.propTypes = {
    name: PropTypes.string.isRequired,
};

const mapStateToProps = ({ users: { name } }) => ({
    name,
});

export default connect(mapStateToProps)(AppComponent);
