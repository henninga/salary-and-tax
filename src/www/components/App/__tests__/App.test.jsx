import mockComponent from 'identity-obj-proxy'
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../index';

jest.mock('../../Users', () => mockComponent);

describe('App', () => {
    it('should render', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
