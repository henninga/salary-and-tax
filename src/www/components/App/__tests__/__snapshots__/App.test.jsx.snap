// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent should render 1`] = `
<div>
  Hello world
</div>
`;
